### ISSUE TITLE
UI: Drop patternfly and use only tailwind (technical debt)

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#1617]
### Is your enhancement related to a problem? Please describe

When implementing patternfly + tailwindcss you have to bounce between both tailwind as well as patternfly to implement a feature.

This causes technical debt with maintaining two different frameworks.

### Describe the solution you'd like

Remove patternfly and use only tailwind.

But "components" and UI templates into the `/ui` folder, etc.

**Comment by deboer-tim** [comment]
Most of the use of PF has been removed over the last few months by creating components and other cleanup. There are now only 68 references in 13 files. We were down 56 references in 11 last week and it grew due to copy/paste of existing code, so would be good to do some final cleanup and complete this.

We're now close enough that I've opened individual issues for the remaining work: 

- #3787
- #3797
- #3799
- #3800
- #3801

Then the final removal from index.html, overrides.css, and removing the dependency.
